src/current/_data/versions.csv

```diff
@@ -12,4 +12,3 @@ v21.2,2021-11-16,2022-11-16,2023-05-16,v21.1
 v22.1,2022-05-24,2023-05-24,2023-11-24,v21.2
 v22.2,2022-12-05,2023-12-05,2024-06-05,v22.1
 v23.1,2023-05-15,2024-05-15,2024-11-15,v22.2
-v23.2,N/A,N/A,N/A,v23.1
```

Comment:
@mdlinville It seems like this row was accidentally deleted. Not sure how that happened. I'll follow up with Ian when he gets back into the office to see if he observed that with the `release-notes.py` script. I don't believe the script does anything to delete rows in `versions.csv`, but I reserve the right to be wrong.

Reply:
I think Ian intentionally removed it based on https://cockroachlabs.slack.com/archives/G01Q35D80EM/p1694729513545189?thread_ts=1694728082.428199&cid=G01Q35D80EM. Should I try putting it back, and revert the changes to the k8s file?

Reply:
I removed this line in order to get `release_notes.py` to work, but I *thought* I reverted the change before pushing.

Reply:
Gotcha. That bug should be fixed, so yes, if someone can just make sure this line gets added back in, we should be golden, and the change in `src/current/v23.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md` can (and should) be reverted.